ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -65,6 +65,10 @@ public NtbsContext(DbContextOptions<NtbsContext> options)
         public virtual DbSet<MBovisOccupationExposure> MBovisOccupationExposures { get; set; }
         public virtual DbSet<MBovisAnimalExposure> MBovisAnimalExposure { get; set; }
         public DbSet<NotificationAndDuplicateIds> NotificationAndDuplicateIds { get; set; }
+        public virtual DbSet<LegacyImportBatchGroup> LegacyImportBatchGroup { get; set; }
+        public virtual DbSet<LegacyImportBatch> LegacyImportBatch { get; set; }
```

Comment:
Why do we have this  entity? The `LegacyImportBatchGroup` represents the migration run (which is the umbrella under which we want to group all this data) but the particular `LegacyImportBatch` seems like it's just an implementation detail. It's a sensible thing to log (for debugging purposes) but do we need it here?

Reply:
Removed as we discussed earlier, we now just have `LegacyImportMigrationRun` class to track a whole run